### ISSUE TITLE
[Trac #175] Is it free for commercial usage?

### ISSUE THREAD

**Issue by s-c-e** [coin-or/Cbc.old#35]
![image](https://user-images.githubusercontent.com/11558962/47100969-69b26f00-d1fe-11e8-8ad1-58dacadabbf0.png)

As you know, Open Solver uses CBC for solving the mathematical model. Can I use COIN-OR CBC (Linear solver) for commercial usage?

I will appreciate your help with this situation.

Best regards,

**Comment by h-g-s** [comment]
yes, the Eclipse license allow the incorporation of CBC in commercial, closed source software.
